site/src/components/tabs.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Tabs
+description: Tabs component in the Atlas Design System
+template: standard
+classType: Component
+classPrefixes:
+  - tabs
+---
+
+# Tabs
+
+Tabs component consists of navigating to the previous or next tab and well and clicking on a tab. The result will change the content in the `tab-item-container` to display the current selected item.
```

Comment:
First off, thank you for writing up documentation. This is a good page to ensure multiple tab items can work at once. I believe as it stands we need a little change. Do each tab nav and each tab item container need to have an idea associated with their group? This way we'll be able to include any number of these elements on the page with no possible overlap?
```txt
[data-tab-nav="id-1"]
[data-tab-items="id-1"]

Reply:
updated